uctMaths/competition/interface/individual_as_template.html

```diff
@@ -2,7 +2,7 @@
     <head>
   
         <style>
-        tr {
+        tr.answersheet {
```

Comment:
BTW, why was this necessary?

Reply:
The confirmation is also html, so if you combine the forms without setting the styles to be for only the answer sheets the confirmation comes out very strangely